src/python/turicreate/toolkits/sound_classifier/sound_classifier.py

```diff
@@ -937,7 +997,7 @@ class as a vector. Label ordering is dictated by the ``classes``
 
         y = []
         for data, in _create_data_iterator(deep_features['deep features'].to_numpy(), None, batch_size=batch_size):
-            y += self._custom_classifier.predict(data).tolist()
+            y += self._custom_classifier.predict(data)[0].tolist()
```

Comment:
Why do we need to subscript here?

Reply:
To remove an extra outer bracket

Reply:
Could we just do this on the TF side? I guess I don't see why we should break compatibility with the MXNet implementation if we don't need to. If you think it makes more sense for predict to return a list, could we add a list to the MXNet side, to keep it in sync?

Reply:
But I think you should ask yourself why you think `predict` should return a list or whatever it is that we need to subscript

Reply:
Removed the subscript since the outer bracket was added because I was running `self.sess.run([self.predictions]` instead of `self.sess.run(self.predictions`. This is fixed now.

The list conversion is happening here to update the `y` list in this code. IMO we shouldn't change `predict` return type based on one specific usage. If we are doing `tolist()` in more than one place I would agree to change the return type of `predict`.

Reply:
Makes sense, I agree!